lib/DatabasePlugins/mongodb.py

```diff
@@ -186,23 +186,29 @@ def query_docs(
                     .find(dict_filter, {"json": 0})
                     .skip(skip)
                 )
-
+        if sort_dir is not None:
+            if sort_dir == "DESC":
+                sort_dir = DESCENDING
+            elif sort_dir == "ASC":
+                sort_dir = ASCENDING
+        else:
+            sort_dir = DESCENDING
```

Comment:
This does not take into account a case where `sort_dir` is set but something else.

This could be either simplified to:

```suggestion
        if sort_dir == "DESC":
            sort_dir = DESCENDING
        elif sort_dir == "ASC":
            sort_dir = ASCENDING
        else:
            sort_dir = DESCENDING
```

Or, if you would like to accept any string beginning with `DESC` or `ASC` including the entire words:

```suggestion
        if isinstance(sort_dir, str):
            if sort_dir.startswith("DESC"):
                sort_dir = DESCENDING
            elif sort_dir.startswith("ASC"):
                sort_dir = ASCENDING
            else:
                sort_dir = DESCENDING
        else:
            sort_dir = DESCENDING
```